parachain/test-parachains/adder/collator/src/lib.rs

```diff
@@ -16,10 +16,10 @@
 
 //! Collator for the adder test parachain.
 
-use codec::{Decode, Encode};
 use futures_timer::Delay;
 use polkadot_node_primitives::{Collation, CollatorFn};
 use polkadot_primitives::v1::{CollatorId, CollatorPair, PoV};
+use parity_scale_codec::{Encode, Decode};
```

Comment:
It seems to be a pattern to rename `parity_scale_codec` to `codec` in the `Cargo.toml` - not sure why, I don't really see much pros or cons.

Reply:
Yeah; afaict that was a change upstream that got pulled in when I merged `master` just now. I have a mild preference for using the full name, but it's not strong enough that I would have made the change on my own.

Reply:
FYI: I removed all (hopefully) renames for parity-scale-codec in #1946.

Reply:
Alright, in that case, ignore.